pkg/image/source.go

```diff
@@ -125,6 +131,7 @@ func detectSource(fs afero.Fs, userInput string) (Source, string, error) {
 					// the docker daemon exists and is accessible
 					return DockerDaemonSource, userInput, nil
 				}
+				log.Errorf("failed to ping docker daemon: %v", err)
```

Comment:
our latest convention is that `log.Error` is reserved for when something goes wrong that is not recoverable (results in a premature exit). In cases like this where we continue with normal execution we should use `log.Warn*`.